luigi/configuration.py

```diff
@@ -49,7 +49,10 @@ class LuigiConfigParser(ConfigParser):
         'luigi.cfg',
     ]
     if 'LUIGI_CONFIG_PATH' in os.environ:
-        _config_paths.append(os.environ['LUIGI_CONFIG_PATH'])
+        if not os.path.isfile(os.environ['LUIGI_CONFIG_PATH']):
+            warnings.warn("LUIGI_CONFIG_PATH is not a valid path to configuration file. Invalid file: {path}".format(path=os.environ['LUIGI_CONFIG_PATH']))
```

Comment:
Should you say that the path is not valid? That sounds like the path-syntax is malformed. Isn't it better to simply say that the file doesn't exit?